clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1037,6 +1536,9 @@ linkAndWrapDeviceFiles(SmallVectorImpl<OffloadFile> &LinkerInputFiles,
 
   std::mutex ImageMtx;
   DenseMap<OffloadKind, SmallVector<OffloadingImage>> Images;
+  // Create a binary image of each offloading image and embed it into a new
+  // object file.
+  SmallVector<StringRef> WrappedOutput;
```

Comment:
I wonder if we need some kind of `TODO`s in this function to be more integrated into existing linker flow, like use existing `wrapDeviceImages` function instead of "wrapping" SYCL device image earlier in the pipeline

Reply:
I will have a better idea of 'device code wrapping' once we move away from clang-offload-wrapper tool. I will try to add a TODO at a later stage.

Thanks